src/PluginForm/Onsite/PaymentMethodAddForm.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\commerce_heartland\PluginForm\Onsite;
+
+use Drupal\commerce_payment\PluginForm\PaymentMethodAddForm as BasePaymentMethodAddForm;
+use Drupal\Core\Form\FormStateInterface;
+
+class PaymentMethodAddForm extends BasePaymentMethodAddForm
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildCreditCardForm(array $element, FormStateInterface $form_state)
+    {
+        $plugin = $this->plugin;
+
+        $element['#attributes']['class'][] = 'heartland-form';
+
+        $element['#attached']['drupalSettings']['heartland'] = [
+            'publicKey' => $plugin->getPublicKey(),
+        ];
+        $element['#attached']['library'][] = 'commerce_heartland/secure-submit';
+        $element['#attached']['library'][] = 'commerce_heartland/form';
```

Comment:
Does this map to our `js/form.js` file? If so, do we need to explicitly define `commerce_heartland/secure-submit` if `commerce_heartland/form` lists `commerce_heartland/secure-submit` as a dependency in the `commerce_heartland.libraries.yml` file?

Reply:
That's simply telling Drupal that form requires secure submit.